tests/test_cli.py

```diff
@@ -73,22 +73,22 @@ def test_get_example_manifest_default(self, runner, helpers, config, data_model_
     # get manifest as a csv
     # use google drive to export
     @pytest.mark.google_credentials_needed
-    def test_get_example_manifest_csv(self, runner, helpers, config, data_model_jsonld):
+    def test_get_example_manifest_csv(self, runner, helpers, config: Configuration, data_model_jsonld):
         output_path = helpers.get_data_path("test.csv")
 
         result = runner.invoke(
-            manifest, ["--config", config.CONFIG_PATH, "get",  "--data_type", "Patient", "--jsonld", data_model_jsonld, "--output_csv", output_path]
+            manifest, ["--config", config.config_path, "get",  "--data_type", "Patient", "--jsonld", data_model_jsonld, "--output_csv", output_path]
         )
         assert result.exit_code == 0
         self.assert_expected_file(result, output_path)
 
     # get manifest as an excel spreadsheet
     @pytest.mark.google_credentials_needed
-    def test_get_example_manifest_excel(self, runner, helpers, config, data_model_jsonld):
+    def test_get_example_manifest_excel(self, runner, helpers, config: Configuration, data_model_jsonld):
         output_path = helpers.get_data_path("test.xlsx")
 
         result = runner.invoke(
-            manifest, ["--config", config.CONFIG_PATH,  "get",  "--data_type", "Patient", "--jsonld", data_model_jsonld, "--output_xlsx", output_path]
+            manifest, ["--config", "/home/alamb/repos/schematic/tests/data/test_config.yml",  "get",  "--data_type", "Patient", "--jsonld", data_model_jsonld, "--output_xlsx", output_path]
```

Comment:
should this be `config.config_path`?

Reply:
Yes, that's right. Thank you for catching that.